### ISSUE TITLE
Find a partner

### ISSUE THREAD

**Issue by pixelass** [pixelass/my-fucked-up-self#1]
It's been 10 long years since I last touched another person. I think this feature would greatly improve the quality of this project called "life".

**Comment by pixelass** [comment]
How the fuck does this work. Where do I even start?

**Comment by Haroenv** [comment]
Dating apps sometimes work for me

**Comment by pixelass** [comment]
I don't believe in dating online or via apps.  
These things can be quite depressing.  
90% of all women consider me too small. It really makes me feel like being a product in a catalog. 

Also: I've wasted over 1000€ in the last year for online dating. Zero success rate

**Comment by mischah** [comment]
> Also: I've wasted over 1000€ in the last year for online dating. Zero success rate

What the actual fuck?
Hard to believe.

Okay. Just throwing in an idea: Have you thought about speed dating?
Might be weird and arkward … plus I don't have a clue which kind of females are attending this. But it might be worth a try.

**Comment by pixelass** [comment]
Yes the actual fuck. Online dating companies are fucking assholes. They did a big part in making me feel like waste.

They just steal your money and tell you to fuck off. 

And all the women in those portals are materialistic. No women ever showed interest. Nobody even replied when I wrote them. I got very few answers of women who told me that they're looking for men that are at lest 10cm taller than them? You think women would know better than discriminating others.... hell no

**Comment by pixelass** [comment]
@mischah I am scared to even try speeddating or don't even know how this works. The problem with depressions is that you're just not really interesting to others. A lot of people feel bad for you but nobody loves you because of it.

Speeddating sounds like opening up to a lot of people fast and then going home to kill yourself I never know what to say to others. Related to #6 

Also related to #22

**Comment by mischah** [comment]
Okay. Doesn’t sound like a good Idea 😔
